extensions/interactions/MultipleChoiceInput/directives/oppia-interactive-multiple-choice-input.component.ts

```diff
@@ -103,32 +101,11 @@ export class InteractiveMultipleChoiceInputComponent implements OnInit {
       }
       return choices;
     };
-    // If choices need to be shuffled, shuffle them, if question is answered
-    // once before, get the previous order of choices, otherwise order the
+    // If choices need to be shuffled, shuffle them, otherwise order the
     // choices based on their original index.
-    this.questionIsAnsweredOnce = this.isQuestionOnceAnswered();
```

Comment:
Why is all of this removed?

Reply:
Ah, I've reverted a PR that caused a bug. See https://github.com/oppia/oppia/pull/14777#issuecomment-1226151784. I've also updated the PR description to mention it.